src/types.ts

```diff
@@ -2146,3 +2146,15 @@ export type Campaign = {
   updated_at: string;
 } & CampaignData &
   CampaignStatus;
+
+export type TaskStatus = {
+  created_at: string;
+  status: string;
+  task_id: string;
+  updated_at: string;
+  error?: {
+    description: string;
+    type: string;
+  };
+  result?: object;
```

Comment:
could this get expanded a bit more?

Reply:
so this result in of an undefined type/shape, what do you suggest the typedef should be in that case?

Reply:
it's either `undefined` or an unknown object? in this case:
```suggestion
  result?: UnknownType;
```